lib/src/cognito_credentials.dart

```diff
@@ -30,14 +32,14 @@ class CognitoCredentials {
     if (expireTime == null ||
         DateTime.now().millisecondsSinceEpoch > expireTime - 60000) {
       final identityId = CognitoIdentityId(_identityPoolId, _pool);
-      final identityIdId = await identityId.getIdentityId(token, authenticator);
```

Comment:
Hi, thanks for contribution,
Could you please explain why did you remove `authenticator` from getIdentityId method? It's optionally used there.

Reply:
Ah, there was a merge conflict and I probably just missed it. Will get that resolved.

Reply:
@furaiev Should be good to go now, sorry about that.

Reply:
It's ok, thank you! :)